presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -355,6 +359,9 @@ private static Expression toStringLiteral(Type type, double value)
             if (type.equals(DATE)) {
                 return new StringLiteral(LocalDate.ofEpochDay(round(value)).toString());
             }
+            if (type.equals(TIMESTAMP)) {
+                return new StringLiteral(new SqlTimestamp(round(value) / MICROSECONDS_PER_MILLISECOND, session.getSqlFunctionProperties().getTimeZoneKey(), MILLISECONDS).toString());
```

Comment:
@gupteaj, can we call SqlTimestamp(long, TimeUnit) instead? That would set the timezone to an empty Optional instance and as far as I can tell SqlTimestamp.toString() doesn't use timezone.